app/models/person_tags/validation_tagged.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module PersonTags::ValidationTagged
+
+  extend ActiveSupport::Concern
+
+  included do
+    after_update :remove_validation_tags
+  end
+
+  private
+
+  # if person can be updated, all validations passed. so we asume it's save to remove
+  # all validation tags which marks this person for having invalid attributes
+  def remove_validation_tags
+    tags = PersonTags::Validation.list
+    if tags.present?
+      ActsAsTaggableOn::Tagging.where(taggable: self, tag: tags).destroy_all
+    end
```

Comment:
This circumvents the permission system in the sense that a Person normally cannot edit their own tags, but I think it is okay here. Just something to be aware of.